doc/ref_model/chapters/chapter10.md

```diff
@@ -69,6 +69,6 @@ Closed-loop automation is evolving as a major advancement in the telecommunicati
 ### 10.3.5 Acceleration Abstraction
 Many vRAN and some other domain's network functions require accelerators to meet latency and thruput requirements. A large number of advanced ASICs, FPGAs, GPUs, and Smart NIC have come to the market to address these needs but unfortunately there is a lack of a common way to utilize them.
 		
-oRAN Alliance is developing a common abstraction and programming model for RAN application domain (see https://oranalliance.atlassian.net/wiki/download/attachments/872841331/O-RAN.WG6.AAL-GAnP-v01.00.pdf?api=v2). P4 Language Consortium (https://p4.org/) defines specifications for switching and routing domains. There is, however, a lack of a common programming model for accelerator applications development.
+O-RAN Alliance (https://www.o-ran.org) is developing a common abstraction and programming model for RAN application domain (see https://www.o-ran.org/specificationsO-RAN document for Acceleration Abstraction Layer General Aspects an Principles 1.0 - November 2020 (O-RAN.WG6.AAL-GAnP-v01.00)). P4 Language Consortium (https://p4.org/) defines specifications for switching and routing domains. There is, however, a lack of a common programming model for accelerator applications development.
```

Comment:
URL is not correct in
https://www.o-ran.org/specificationsO-RAN document for Acceleration Abstraction Layer General Aspects an Principles 1.0 - November 2020 (O-RAN.WG6.AAL-GAnP-v01.00)
and could be changed to
[O-RAN document for Acceleration Abstraction Layer General Aspects an Principles 1.0 - November 2020 (O-RAN.WG6.AAL-GAnP-v01.00)](https://www.o-ran.org/specifications)
.

Reply:
Petar, how do create a hyperlink with a title in this doc?

Reply:
I think latest patch resolves it.

Reply:
@arkadykanevsky : Yes, looks good now.